.github/workflows/publish.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: publish docker image
         uses: hypertrace/github-actions/gradle@main
         with: 
-          args: publish dockerPushImages
```

Comment:
Because HTC is not currently supported as its own deployment, it may be worth removing the entire publish workflow. Up to you.

Reply:
Let's keep it for now. Because if that caught up again then we have to fix it anyways.